about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Muson 2021</title>
+  <script src="https://kit.fontawesome.com/2cfd3e74c6.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+  <link rel="stylesheet" href="./CSS/style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&amp;display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header id="header">
+    <nav class="first-navbar bg-dark color-white-text">
+      <div class="fa-icon">
+        <a href="#"><i class="fab fa-facebook-f" aria-hidden="true"></i></a>
+        <a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a>
+      </div>
+      <ul class="small-nav-bar">
+        <li><a href="#">Logout</a></li>
+        <li><a href="#">My page</a></li>
+        <li><a href="#">English</a></li>
+      </ul>
+    </nav>
+    <button class="menu-toggle" type="button">
+      <img src="images/menu-hambuger1.svg" alt="menu hamburger" aria-hidden="true">
+    </button>
+    <nav class="second-navbar">
+      <a href="./index.html" class="muson-logo"><img src="images/home-logo.jpg" alt="muson logo"></a>
+      <ul class="second-nav-items">
+        <li><a class="nav-links" href="./about.html">About</a></li>
```

Comment:
- [ ] kindly add a link to navigate to the `home page` instead of having an about here🤷